### PR TITLE
Add XOR function to calculator

### PR DESCRIPTION
**Pull request by penpaperkeycode**
Add xor calculate function code

### CODE REVIEW

**Comment by IntelligenceDatum** [comment]
Please review my request :)

**Review by seungjaeryanlee — DISMISSED** [review]
`.idea/` folder should not be included in the PR.

**Comment by Curt-Park** [comment]
Please fix the PR name.

**Comment by IntelligenceDatum** [comment]
> `.idea/` folder should not be included in the PR.

Thanks for notice! I deleted it

**Comment by IntelligenceDatum** [comment]
> Please fix the PR name.

Yes master, I changed PR title.

**Comment by Curt-Park** [comment]
Please check the code. You just pushed the conflicting lines.

**Comment by IntelligenceDatum** [comment]
> Please check the code. You just pushed the conflicting lines.

Master, I've update the code agian. plz check my humble code.

File: test_calculator.py [hunk 1]
```diff
@@ -16,6 +16,12 @@ def test_add(self):
         pass
 
     def test_sub(self):
+        """Put your test cases for sub that starts with assert."""
```
Comment (Curt-Park) [inline test_calculator.py]: Please add a description for the test cases.

Comment (penpaperkeycode) [reply test_calculator.py]: Add Done! 

How can i solve the ryans query above?

**Review by Curt-Park — APPROVED** [review]
Good job. Looks good to me.

**Comment by penpaperkeycode** [comment]
> `.idea/` folder should not be included in the PR.

Please review my code again.

**Review by seungjaeryanlee — CHANGES_REQUESTED** [review]
Please fix the typo in the PR title: the ^ operator is the XOR operator, not the XNOR operator. Otherwise, the PR looks good!

**Comment by Curt-Park** [comment]
@seungjaeryanlee I missed that point! Great review!

**Comment by penpaperkeycode** [comment]
> Please fix the typo in the PR title: the ^ operator is the XOR operator, not the XNOR operator. Otherwise, the PR looks good!

My bad! Thank you :)

**Review by seungjaeryanlee — APPROVED** [review]
Looks good! :+1:

**Pull request merged**